overlord/servicestate/quota_control.go

```diff
@@ -479,26 +479,28 @@ func UpdateQuota(st *state.State, name string, updateOpts QuotaGroupUpdate) erro
 	return ensureSnapServicesForGroup(st, grp, allGrps, nil)
 }
 
-// RemoveSnapFromQuota removes the specified snap from the quota.
-// XXX: this should go away and just become an option to UpdateQuota but that is
-// more complexity than we can handle right now
-func RemoveSnapFromQuota(st *state.State, group, snap string) error {
+// EnsureSnapAbsentFromQuotaGroup ensures that the specified snap is not present
+// in any quota group, usually in preparation for removing that snap from the
+// system to keep the quota group itself consistent.
+func EnsureSnapAbsentFromQuotaGroup(st *state.State, snap string) error {
```

Comment:
no function here uses QuotaGroup in full in their names, so just "EnsureSnapAbsentFromQuota" ?  other options could be:

WithdrawSnapFromQuota
DisplaceSnapFromQuota
RemoveSnapFromQuotaIfAny
RetireSnapFromQuota

Reply:
I went with EnsureSnapAbsentFromQuota